nrf_802154/driver/include/nrf_802154.h

```diff
@@ -1185,18 +1215,20 @@ void nrf_802154_src_addr_matching_method_set(nrf_802154_src_addr_match_t match_m
  * To better illustrate, if RSSI is to be inserted into ACKs for specific address,
  * following ie data needs to be prepared:
  *
+ * @verbatim
  * +------------+----------------------+---------------------------------+-----------------------------+
  * | Bytes: 0-1 |          2-4         |                5                |              6              |
  * +------------+----------------------+---------------------------------+-----------------------------+
  * | IE header  | IE_VENDOR_THREAD_OUI | IE_VENDOR_THREAD_ACK_PROBING_ID | IE_VENDOR_THREAD_RSSI_TOKEN |
  * +------------+----------------------+---------------------------------+-----------------------------+
  *                                     |                                                               |
  *                                     | <------------------IE Vendor-specific data------------------> |
+ * @endverbatim
  *
  * When sending ACK with this data, before transmission, RSSI of the last received frame
  * will be written into byte 6.
  *
- * The method can be set during initialization phase by calling @ref nrf_802154_src_matching_method.
+ * The method can be set during initialization phase by calling @ref nrf_802154_ack_data_src_addr_matching_method_set.
```

Comment:
Shouldn't this be `nrf_802154_src_addr_matching_method_set` instead? The function referenced now is not part of public API. Same applied to other occurrences.